src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -263,6 +263,7 @@ public void bannedUserCanNotEnterTheChat() {
     ChatPage chatUserToBeBanned = openChatForUser(userToBeBanned, userToBeBannedPassword);
 
     switchToWindow(0);
+    String name = chatUserStaff.getUserName();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "name" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)